cpp/cmake/Dependencies.cmake

```diff
@@ -38,8 +38,8 @@ else(DEFINED ENV{RAFT_PATH})
   set(RAFT_DIR ${CMAKE_CURRENT_BINARY_DIR}/raft CACHE STRING "Path to RAFT repo")
 
   ExternalProject_Add(raft
-    GIT_REPOSITORY    https://github.com/rapidsai/raft.git
-    GIT_TAG           3f8a4bf6f81289f1fdaae9a5bd4a10de8674aa5c
+    GIT_REPOSITORY    https://github.com/wphicks/raft.git
+    GIT_TAG           fea-move-distancetype
```

Comment:
Presumably the RAFT merge happens first so we eliminate this?

Reply:
Yes, that's right. Once the RAFT merge is in, I'll switch this back to the corresponding pinned commit on the main repo. Maybe we should have a specific PR label for this situation so we don't accidentally merge something like this in early?